docs/content/Coding Milestones/PoC2023q1/mailbox-controller.md

```diff
@@ -135,13 +135,14 @@ In a separate terminal window(3), create an inventory management workspace as fo
 ```shell
 kubectl ws \~
 kubectl ws create imw --enter
+kubectl kcp bind apiexport root:espw:edge.kcp.io
```

Comment:
This added command should also get added text talking about it.

Reply:
ok